marketplace/vaultwarden.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "./schema.json",
+    "code": "vaultwarden",
+    "name": "vaultwarden",
+    "description": "A Bitwarden compatible server written in Rust.",
+    "icon": "https://github.com/dani-garcia/vaultwarden/raw/main/resources/vaultwarden-icon.svg",
```

Comment:
I think [this](https://github.com/dani-garcia/vaultwarden/raw/main/resources/vaultwarden-icon-white.svg) white one may look better in our dark dashboard?